interface/billing/sl_eob_search.php

```diff
@@ -188,11 +190,21 @@ function upload_file_to_client_pdf($file_to_send) {
         $stmt['cid'] = $row['pid'];
         $stmt['pid'] = $row['pid'];
 		$stmt['dun_count'] = $row['stmt_count'];
-		$stmt['bill_note'] = $row['billing_note'];
+		$stmt['bill_note'] = $row['pat_billing_note'];
+        $stmt['enc_bill_note'] = $row['enc_billing_note'];
 		$stmt['bill_level'] = $row['last_level_billed'];
 		$stmt['level_closed'] = $row['last_level_closed'];
         $stmt['patient'] = $row['fname'] . ' ' . $row['lname'];
+		#If you add a field to the demographics layout called 
+		#in_care_of this will allow you to send statements to the parent
+		#of a child or a guardian etc
+        if(strlen($row['in_care_of']) == 0) {
```

Comment:
Should this instead be the Guardian name field that is already there?
(note there is now en entire Guardian tab in demographics which was required by MU2)